extensions/positron-assistant/src/test/anthropic.test.ts

```diff
@@ -58,61 +59,24 @@ suite('AnthropicLanguageModel', () => {
 		sinon.restore();
 	});
 
-	test('provideLanguageModelResponse should filter out messages with empty text content', async () => {
```

Comment:
I removed this test case because it is covered in the `provideLanguageModelResponse filters empty messages and different LanguageModelTextPart contents correctly` test